CHANGELOG.rst

```diff
@@ -4,6 +4,7 @@ Changelog
 in development
 --------------
 
+* Removed support for medium-strength ciphers from default nginx configuration (#3244)
```

Comment:
Thanks. That reminds me, I've got a proposal for a new structure for the changelogs. I'll submit a RFR PR